src/components/MetricsOptions/__tests__/MetricsOptionsBar.test.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { mount, shallow } from 'enzyme';
+
+import MetricsOptionsBar from '../MetricsOptionsBar';
+
+const optionsChanged = jest.fn();
+const lastOptionsChanged = () => {
+  return optionsChanged.mock.calls[optionsChanged.mock.calls.length - 1][0];
+};
+
+describe('MetricsOptionsBar', () => {
+  it('renders initial layout', () => {
+    const wrapper = shallow(<MetricsOptionsBar onOptionsChanged={jest.fn()} />);
+    const eltDuration = wrapper.find('#duration');
+    expect(eltDuration.length).toBe(1);
+    const eltStep = wrapper.find('#step');
+    expect(eltStep.length).toBe(1);
+    const eltRateInterval = wrapper.find('#rateInterval');
+    expect(eltRateInterval.length).toBe(1);
+  });
+
+  it('changes trigger parent callback', () => {
```

Comment:
This test is interesting.
There are not enough tests in any project.
My only comment is about the  tradeof of this tests in the UI.
Current UI is ephemeral as is going to be changed as soon as we review basic UX flows, so perhaps these tests have a very low granilarity.
If these can be reused, it's fine, but I think with a 'renders initial layout' would be enough for the stage we are.